ecsworkshop/task_definition.json

```diff
@@ -6,7 +6,7 @@
         "logDriver": "awslogs",
         "options": {
           "awslogs-group": "{{LOGGROUPNAME}}",
-          "awslogs-region": "us-west-2",
+          "awslogs-region": "((AWS_REGION}}",
```

Comment:
at first glance, is this a typo? open with parens, close with curly braces?

Reply:
typo - good catch! I'll fix it

Reply:
cool! I'll have to defer to @adamjkeller as to whether or not the value is available to expand.

Reply:
Thanks! Yeah, so this looks like a bug :-) In the workshop we are doing a search and replace on the `task_definition.json` to replace with the desired values. In this instance, we are searching for {{REGION}} and replacing with AWS_REGION. Can you change the value inside to be `{{REGION}}`?

Here's the sed command we run to replace the values:

https://ecsworkshop.com/stateful_workloads/deployapplication/

```
  sed "s|{{EXECUTIONROLEARN}}|$execution_role_arn|g;s|{{TASKROLEARN}}|$task_role_arn|g;s|{{FSID}}|$fs_id|g;s|{{LOGGROUPNAME}}|$log_group_name|g;s|{{REGION}}|$AWS_REGION|g" task_definition.json > task_definition.automated
```